perfkitbenchmarker/linux_benchmarks/aws_dynamodb_ycsb_benchmark.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Run YCSB benchmark against AWS DynamoDB.
+
+This benchmark does not provision VMs for the corresponding DynamboDB database.
+The only VM group is client group that sends requests to specifiedDB.
+
+Before running this benchmark, you have to manually create `usertable` with primaryKey 'user01'.
```

Comment:
It looks like AWS provides a way to create a table via command-line flags. What would it take to create this via the command-line from PKB?

Reply:
it would take a quick boto3 command to create and destroy the table, but that doesn't take into consideration things like autoscaling/global tables/size of RCU/WCU provisioning.

Reply:
Can we specify those via a command too?

For setting up elasticache for redis on AWS, we configure the cache node type and replication via the command-line tool. See:

https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/providers/aws/aws_elasticache_redis.py

If we can configure dynamodb from the commandline, this will make it easier for others to run this benchmark.

We can also add a TODO to the code to consider this at a later point, or just setup dynamodb with some default configurations and add the knobs later.

Reply:
i got most of the code for configuring the server, ill look at the provider format.